matscipy/dftbp_socketcalc.py

```diff
@@ -0,0 +1,220 @@
+# ======================================================================
+# matscipy - Python materials science tools
+# https://github.com/libAtoms/matscipy
+#
+# Copyright (2014) James Kermode, King's College London
+#                  Lars Pastewka, Karlsruhe Institute of Technology
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+# ======================================================================
+
+"""The dftbsocket module.
+
+This python module provides an ASE calculator for DFTB+.
+It operates DFTB+ through the i-PI interface protocol, which is implemented in DFTB+.
+i-PI can be obtained from https://github.com/i-pi/i-pi/tree/master.
+The i-PI dependency is not (yet) compatible with python 3,
+so the DFTB+ wrapper will not work with python 3 until i-PI is fixed.
```

Comment:
Why is this the case? Even if i-PI is not Python 3, the socket calculator part is probably Fortran? I guess there is a patch for DFTB+ at that location. Please update documentation that this becomes clear.